nautobot/extras/models/tags.py

```diff
@@ -80,9 +80,28 @@ def validate_content_types_removal(self, content_types_id):
         return errors
 
 
+class TaggedItemManager(BaseManager.from_queryset(RestrictedQuerySet)):
+    def get_by_natural_key(self, *args):
+        return self.get(
+            tag__name=args[0],
+            content_type__app_label=args[1],
+            content_type__model=args[2],
+            object_id=args[3],
+        )
```

Comment:
Instead of indexing here with positional args, can we please use named positional arguments?
```suggestion
    def get_by_natural_key(self, tag_name, app_label, model, object_id):
        return self.get(
            tag__name=tag_name,
            content_type__app_label=app_label,
            content_type__model=model,
            object_id=object_id,
        )
```